edge-hub/src/Microsoft.Azure.Devices.Routing.Core/endpoints/statemachine/EndpointExecutorFsm.cs

```diff
@@ -81,6 +81,7 @@ public class EndpointExecutorFsm : IDisposable
         readonly Timer retryTimer;
         readonly AsyncLock sync = new AsyncLock();
         readonly ISystemTime systemTime;
+        readonly CancellationToken endpointExecutorToken;
```

Comment:
>endpointExecutorToken [](start = 35, length = 21)

no need to denote that this comes from the executor, right?  It can prob just be named cancellationToken

Reply:
+1